### PR TITLE
Proposal: remove double checking CI after merge to master / release

### PR DESCRIPTION
**Pull request by MrDarthShoe**
image

### CODE REVIEW

**Comment by evik42** [comment]
After CI builds run correctly on the PR, master branch can have new commits so it needs to be checked after merge.

**Comment by MrDarthShoe** [comment]
> After CI builds run correctly on the PR, master branch can have new commits before the actual merge happens. It needs to be checked after merge.

When new commits arrive to master, PR should be out-of-date (which will disable merge button) and CI should be ran on PR again with new commits on master

**Review by evik42 — APPROVED** [review]
I agree

**Pull request merged**
